### PR TITLE
Pytest 2.8.3

### PR DESCRIPTION
**Pull request by RonnyPfannschmidt**
please check the changelog grooming and the cross-ceck the announcement

i will merge this one myself after the release is done

### CODE REVIEW

File: doc/en/announce/release-2.8.3.rst [hunk 1]
```diff
@@ -0,0 +1,56 @@
+pytest-2.8.3: bug fixes
+=======================
+
+pytest is a mature Python testing tool with more than a 1100 tests
+against itself, passing on many different interpreters and platforms.
+This release is supposed to be drop-in compatible to 2.8.1.
```
Comment (nicoddemus) [inline doc/en/announce/release-2.8.3.rst]: 2.8.1 -> 2.8.2

File: doc/en/announce/release-2.8.3.rst [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+As usual, you can upgrade from pypi via::
+
+    pip install -U pytest
+
+Thanks to all who contributed to this release, among them:
+
+    Bruno Oliveira
+    Demian Brecht
+    Florian Bruhin
```
Comment (nicoddemus) [inline doc/en/announce/release-2.8.3.rst]: This list is missing Gabriel Reis, which contributed the test name highlighting

**Comment by nicoddemus** [comment]
Thanks @RonnyPfannschmidt for making the release.

Question: did you run regendocs?

File: doc/en/announce/release-2.8.3.rst (hunk 2 above)
Comment (The-Compiler) [inline doc/en/announce/release-2.8.3.rst]: I used `git log --pretty='format:%an' 2.8.1..HEAD | sort | uniq` in the past. I'll add that to `HOWTORELEASE`.

File: CHANGELOG [hunk 3]
```diff
@@ -19,6 +19,14 @@
   they stand out in the output.
   Thanks Gabriel Reis for the PR.
 
+- add more talks to the documentation
```
Comment (The-Compiler) [inline CHANGELOG]: Not sure if this is worth a changelog entry (and the next one too), but your decision :wink:

File: doc/en/announce/release-2.8.3.rst (hunk 2 above)
Comment (nicoddemus) [inline doc/en/announce/release-2.8.3.rst]: Thanks @The-Compiler! :smile:

File: CHANGELOG [hunk 4]
```diff
@@ -19,6 +19,14 @@
...
+- extend documentation on the --ignore cli option 
+- use pytest-runner for setuptools integration 
+- minor fixes for interaction with elcapitan system integrity protection (thanks Florian)
```
Comment (The-Compiler) [inline CHANGELOG]: Might want to say "OS X El Capitan" here so it's clear what this means, even I had to double-read it to understand :laughing:

File: doc/en/example/parametrize.rst [hunk 5]
```diff
@@ -397,8 +397,11 @@ is to be run with different sets of arguments for its three arguments:
 Running it results in some skips if we don't have all the python interpreters installed and otherwise runs all combinations (5 interpreters times 5 interpreters times 3 objects to serialize/deserialize)::
 
    . $ py.test -rs -q multipython.py
-   ...........................
-   27 passed in 0.12 seconds
+   ssssssssssss...ssssssssssss
+   ======= short test summary info ========
+   SKIP [12] $REGENDOC_TMPDIR/CWD/multipython.py:22: 'python3.3' not found
+   SKIP [12] $REGENDOC_TMPDIR/CWD/multipython.py:22: 'python2.6' not found
+   3 passed, 24 skipped in 0.12 seconds
```
Comment (The-Compiler) [inline doc/en/example/parametrize.rst]: Seems like you have less python interpreters installed than @hpk42 has :laughing:

File: HOWTORELEASE.rst [hunk 6]
```diff
@@ -8,7 +8,9 @@ Note: this assumes you have already registered on pypi.
 2. Check and finalize CHANGELOG
 
 3. Write doc/en/announce/release-VERSION.txt and include
-   it in doc/en/announce/index.txt
+   it in doc/en/announce/index.txt::
+
+        git log 2.8.2..HEAD --format='%aN' | sort -u # lists the names of authors involved
```
Comment (The-Compiler) [inline HOWTORELEASE.rst]: Ah, you were faster - thanks!

**Comment by RonnyPfannschmidt** [comment]
@nicoddemus i#ll keep using whatever my distro ships for now

im planning to use nix for some details in future

File: doc/en/example/parametrize.rst (hunk 5 above)
Comment (RonnyPfannschmidt) [inline doc/en/example/parametrize.rst]: new system effect :)

**Comment by nicoddemus** [comment]
LGTM :+1:

**Comment by RonnyPfannschmidt** [comment]
ill wait for the ci to finish, then do the pypi uploads and announce mails

**Comment by nicoddemus** [comment]
Finally. :sweat_smile:

**Pull request merged**